resources/assets/tconstruct/lang/zh_cn.lang

```diff
@@ -570,283 +576,283 @@ material.slimeleaf_purple.name=material.slimeleaf_blue.name
-item.tconstruct.shuriken.desc=回旋镖是一个发射速度很快的短程攻击武器，数量多但是攻击力低下\n也可以从副手投掷出去
-item.tconstruct.shortbow.desc=短弓是种远程武器。向敌人快速精准地射出箭矢。\n使用匠魂的箭。
-item.tconstruct.longbow.desc=长弓是一个有力的远程武器。它能提供更高的伤害，但是射程要比短弓短。\n使用匠魂的箭。
-item.tconstruct.arrow.desc=箭矢是弓所用的弹药。一个箭矢可以多次发射，并且它也可以像其他工具一样被强化。
-item.tconstruct.crossbow.desc=十字弓是一个装填速度缓慢但威力强劲的武器。它可以事先通过右键来装填好，之后任意时刻就可以直接发射。\n使用匠魂的弩箭。
-item.tconstruct.bolt.desc=弩箭是十字弓所用的弹药。一个弩箭可以多次发射，并且它也可以像其他工具一样被强化。\n有几率能穿透敌人的装甲。
+item.tconstruct.pickaxe.desc=镐是一种单体采矿工具，能够高效采掘石料和矿石。\n它真的能破坏方块，好吧？
+item.tconstruct.shovel.desc=锹是用于开掘泥土的工具，能够高效开掘泥土、沙子、沙砾和雪。\n请勿自掘坟墓！
+item.tconstruct.hatchet.desc=短柄斧能砍伐木材，也能高效清理树叶，还是一种凑合能用的武器。\n快劈，快！
+item.tconstruct.mattock.desc=鹤嘴锄具备切割类功能，是一种多功能农具，在破坏木材、泥土和植物时都有良好表现，还具备可观的攻击力。\n右击：锄地
+item.tconstruct.kama.desc=短刃镰是一种单体收割工具，可用于收割植物和给动物剪毛。\n右击：收获作物并重新播种
+item.tconstruct.broadsword.desc=阔剑是一种泛用型武器，其横扫攻击能阻截成群结队的敌人。\n还能高效清理蜘蛛网！
+item.tconstruct.longsword.desc=长剑是一种进攻型武器，常用于全速切入战斗。\n右击：冲跃
+item.tconstruct.rapier.desc=细剑是一种特制型武器，依赖快速刺击来战胜对手。\n右击：后跳
+item.tconstruct.frypan.desc=平底锅是一种重型武器，能通过强大的击退能力来拉开战斗距离。还会发出嘣的一声巨响！\n右击：轰飞
+item.tconstruct.battlesign.desc=战斗牌子是一种顶级防御武器，这一武器技术的进步对下界随处可见的僵尸猪人构成了严重威胁。\n也可当作盾牌使用，能反弹弹射物。
+item.tconstruct.hammer.desc=锤是一种范围采矿工具，能大范围地开采方块。\n对亡灵生物有奇效。
+item.tconstruct.lumberaxe.desc=伐木斧是一种范围采伐工具，一斧就能伐倒整棵树，或是砍下大范围的木料。\n放树啦！
+item.tconstruct.excavator.desc=开掘铲是一种范围挖掘工具，能大范围地挖掘大片区域中的泥土和雪。\n地球化！
+item.tconstruct.cleaver.desc=劈刀是一种重型进攻型武器，其斩击沉重凶悍，但也难以操控。\n劈刀的优势在于超高的单次伤害。\n攻击缓慢，需要双手握持！
+item.tconstruct.battleaxe.desc=战斧是一种重型进攻型武器，充满狂暴战士的力量。它能轻松劈倒小树，也能把对手打飞。\n右击：狂暴
+item.tconstruct.scythe.desc=长柄镰是一种范围收割工具，以摧枯拉朽之势收割作物和敌人的生命。\n具备范围攻击能力。\n右击：收获作物并重新播种
+item.tconstruct.shuriken.desc=回旋镖是一种快速、短距离的投掷型武器，弹药数量大但伤害低。\n副手也可投掷。
+item.tconstruct.shortbow.desc=短弓是一种快速而灵活的武器，使用者在快速射箭的同时还能维持迅捷的移动。\n使用匠魂羽箭。
+item.tconstruct.longbow.desc=长弓是一种强劲的长射程武器，比起它较为短小的兄弟，它的伤害更高，但灵活性有所欠缺。\n使用匠魂羽箭。
+item.tconstruct.arrow.desc=羽箭是匠魂弓所使用的弹药，每一组都能支持很多次射击，而且它们也能像常规工具那样被强化。
+item.tconstruct.crossbow.desc=弩弓是一种攻击缓慢但威力巨大的武器，必须提前长按右键来进行装填，但在装填完毕后可随时射击。\n使用匠魂弩箭。
+item.tconstruct.bolt.desc=弩箭是匠魂弩弓所使用的弹药，每一组都能支持很多次射击，而且它们也能像常规工具那样被强化。\n具备一定的穿甲性能。
 
 
 ################
 #   Tooltips   #
 ################
-tooltip.tool.modifiers=剩余强化次数
+tooltip.tool.modifiers=空余强化槽
 tooltip.tool.broken=损坏
 tooltip.tool.empty=空
-tooltip.tool.holdShift=按下§e§oShift§r查看统计数据
-tooltip.tool.holdCtrl=按下§e§bCtrl§r查看详细信息
+tooltip.tool.holdShift=按住§e§oShift键§r§7查看统计数据
+tooltip.tool.holdCtrl=按住§e§bCtrl键§r§7查看详细信息
 
 tooltip.part.material_added_by=材料由%s添加
-tooltip.part.missing_material=缺失材料： %s
-tooltip.part.missing_stats=材料缺失必要的数据：\n%s
-tooltip.part.missing_info=部件没有数据
+tooltip.part.missing_material=缺失材料：%s
+tooltip.part.missing_stats=材料缺失以下必要数据：\n%s
+tooltip.part.missing_info=部件无数据
 
-tooltip.pattern.cost=材料消耗： %d
+tooltip.pattern.cost=材料消耗：%d
 
-tooltip.chest.has_items=包含物品
-tooltip.patternchest.holds_patterns=包含%d个模具
-tooltip.patternchest.holds_casts=包含%d个铸模
+tooltip.chest.has_items=存有%d
```

Comment:
|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
tooltip.chest.has_items=存有%d个物品
```

Reply:
Get it.